frappe/installer.py

```diff
@@ -124,6 +140,52 @@ def install_db(root_login=None, root_password=None, db_name=None, source_sql=Non
 	frappe.flags.in_install_db = False
 
 
+def find_org(org_repo: str) -> Tuple[str, str]:
+	from frappe.exceptions import InvalidRemoteException
+	import requests
+
+	org_repo = org_repo[0]
+
+	for org in ["frappe", "erpnext"]:
+		res = requests.head(f"https://api.github.com/repos/{org}/{org_repo}")
+		if res.ok:
+			return org, org_repo
+
+	raise InvalidRemoteException
+
+
+def fetch_details_from_tag(_tag: str) -> Tuple[str, str, str]:
+	app_tag = _tag.split("@")
+	org_repo = app_tag[0].split("/")
+
+	try:
+		repo, tag = app_tag
+	except ValueError:
+		repo, tag = app_tag + [None]
+
+	try:
+		org, repo = org_repo
+	except Exception:
+		org, repo = find_org(org_repo)
+
+	return org, repo, tag
+
+
+def parse_app_name(name: str):
+	name = name.rstrip("/")
```

Comment:
Add docstrings & bench ref so that we're mindful of any upcoming changes. 

Ideally, this should be a separate library...maybe we'll do that when this matures & grows more.